lib/iphone-simulator-xcode-7.ts

```diff
@@ -99,6 +100,34 @@ export class XCode7Simulator implements ISimulator {
 		return this.simctl.getAppContainer(deviceId, applicationIdentifier);
 	}
 
+	public getInstalledApplications(deviceId: string): IFuture<IApplication[]> {
+		return common.getInstalledApplications(deviceId);
+	}
+
+	public installApplication(deviceId: string, applicationPath: string): IFuture<void> {
+		return this.simctl.install(deviceId, applicationPath);
+	}
+
+	public uninstallApplication(deviceId: string, appIdentifier: string): IFuture<void> {
+		return this.simctl.uninstall(deviceId, appIdentifier, {skipError: true});
+	}
+
+	public startApplication(deviceId: string, appIdentifier: string): IFuture<string> {
+		return this.simctl.launch(deviceId, appIdentifier);
+	}
+
+	public stopApplication(deviceId: string, appIdentifier: string): IFuture<string> {
+		try {
+			return childProcess.exec(`killall ${appIdentifier.split(".")[2]}`, {skipError: true});
+		} catch(e) {
+			//this.$logger.trace("Unable to kill simulator: " + e);
```

Comment:
Consider uncommenting or removing this code. Also, in messages shown to the user, consider using more neutral terms like "stop".